components/brave_wallet/browser/fil_tx_manager.cc

```diff
@@ -65,20 +213,31 @@ void FilTxManager::RetryTransaction(const std::string& tx_meta_id,
 void FilTxManager::GetTransactionMessageToSign(
     const std::string& tx_meta_id,
     GetTransactionMessageToSignCallback callback) {
-  NOTIMPLEMENTED();
+  std::unique_ptr<FilTxMeta> meta =
+      GetFilTxStateManager()->GetFilTx(tx_meta_id);
+  if (!meta) {
+    VLOG(1) << __FUNCTION__ << "No transaction found with id:" << tx_meta_id;
+    std::move(callback).Run(absl::nullopt);
+    return;
+  }
+  auto message = meta->tx()->GetMessageToSign();
+  auto encoded = brave_wallet::ToHex(message);
+  std::move(callback).Run(encoded);
```

Comment:
Please leave this part in the transaction signing PR, I'm not sure how to review this right now.

Reply:
ok, removed